addons/core/functions/fnc_radioReplaceProcess.sqf

```diff
@@ -19,32 +19,24 @@
         [TFAR_fnc_radioReplaceProcess, 2] call CBA_fnc_addPerFrameHandler;
 */
 
-if !(TF_use_saved_sw_setting) then {
```

Comment:
This file shouldn't do anything when CBA settings are not there yet.
This is the place to wait at to not initialize SR radios.